api/swagger-spec/autoscaling_v1.json

```diff
@@ -291,7 +294,8 @@
       ],
       "produces": [
        "application/json",
-       "application/vnd.kubernetes.protobuf"
+       "application/vnd.kubernetes.protobuf",
```

Comment:
hmm - I wondering why in some api calls we are adding:
"application/json;stream=watch"
and in others:
"application/vnd.kuberentes.protobuf;stream=watch".

Shouldn't these 2 come together?